d2l-image.js

```diff
@@ -1,81 +1,114 @@
-import '@polymer/polymer/polymer-legacy.js';
-import '@polymer/iron-ajax/iron-ajax.js';
-import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
-const $_documentContainer = document.createElement('template');
-
-$_documentContainer.innerHTML = `<dom-module id="d2l-image">
-	<template strip-whitespace="">
-		<style>
+import 'd2l-fetch/d2l-fetch.js';
+import { css, html, LitElement, nothing } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+class D2LImage extends LitElement {
+	static get properties() {
+		return {
+			/**
+			 * Alternate text describing the image
+			 * @type {string}
+			 */
+			alternateText: { type: String, attribute: 'alternate-text' },
```

Comment:
The old property reflected to the attribute, but I don't think it was necessary so seems fine to not do that.

Reply:
Hm, I can reflect here if you think it would be valuable 🤔

Reply:
Nah, just wanted to point it out in case you notice a regression later -- should be fine though.